pkg/operator/ceph/object/spec.go

```diff
@@ -319,8 +319,11 @@ func (c *clusterConfig) reconcileService(cephObjectStore *cephv1.CephObjectStore
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to create or update object store %q service", cephObjectStore.Name)
 	}
-
-	logger.Infof("ceph object store gateway service running at %s:%d", svc.Spec.ClusterIP, cephObjectStore.Spec.Gateway.Port)
+	if cephObjectStore.Spec.Gateway.SecurePort != 0 && cephObjectStore.Spec.Gateway.SSLCertificateRef != "" {
```

Comment:
Nit: IMHO store the value of port into variable here instead of having log message and add the log message after if condition with port value

Reply:
Done.